.github/workflows/build-test.yml

```diff
@@ -0,0 +1,56 @@
+name: GitHub CI
+
+on:
+  push:
+    branches:
+      - kb/master_fast_test
+
+env:
+  REPO_NAME: ${{ github.repository }}
+  EVENT_NAME: ${{ github.event_name }}
+
+jobs:
+
+  linux-test:
+    name: Test 3.7 Linux
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Linux
+        run: |
+          set -x -e
+          bash -x -e .github/workflows/build.space.sh
+          bash -x -e tests/test_pulsar/pulsar_test.sh
+          bash -x -e tests/test_kafka/kafka_test.sh
+          bash -x -e tests/test_aws/aws_test.sh
+          bash -x -e tests/test_gcloud/test_pubsub_bigtable.sh
```

Comment:
this starts a bigtable emulator in the background. I tried using that as well, but with no success. tests still fail